pkg/image/api/validation/validation_test.go

```diff
@@ -488,7 +488,7 @@ func TestValidateImageStream(t *testing.T) {
 				},
 			},
 			expected: field.ErrorList{
-				field.Invalid(field.NewPath("spec", "tags").Key("badid").Child("from", "name"), "abc@badid", "only tags can be scheduled for import"),
```

Comment:
It looks like we lost a test case where we validate that you can't specify a digest for import where the error returned is "only tags can be scheduled for import"